_data/people/VourMa.yml

```
@@ -0,0 +1,19 @@
+active: true
+focus-area:
+  - ia
+institution: University of California, San Diego
+name: Emmanouil Vourliotis
+photo: "/assets/images/team/Emmanouil-Vourliotis.jpg"
+shortname: VourMa
+title:
+website: https://github.com/VourMa
+presentations:
+  - title: Speeding up the CMS track reconstruction with a parallelized and vectorized Kalman-filter-based algorithm during the LHC Run 3 (poster)
+    meeting: ACAT 2022
+    meetingurl: https://indico.cern.ch/event/1106990/
+    url: https://indico.cern.ch/event/1106990/contributions/4998120/attachments/2533387/4359448/ACAT2022_MkFit.pdf
```

```suggestion
    url: https://indico.cern.ch/event/1106990/contributions/4998120/
```

It is preferable to not link directly to PDFs but to the Indico contribution.